src/language/CSSUtils.js

```diff
@@ -62,6 +62,16 @@ define(function (require, exports, module) {
         _invertedBracketPairs = _.invert(_bracketPairs);
     
     /**
+     * Determines if file extension is a CSS preprocessor file extension that Brackets supports.
```

Comment:
Now that we've made the coupling more explicit, maybe change to "...that CSSUtils supports"?

Actually, the references to file extension here are misleading since `getLanguageForPath()` should be called with a fullPath whenever possible, and afaict current callers do always pass a path.  So maybe reword the whole thing to: "Determines if the given path is a CSS preprocessor file that CSSUtils supports."  And remove the "file name" part from the `@param` below.